PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Pull request title
+
+<!-- Open this PR as draft while it is not ready -->
+<!-- Use keywords like 'close' or 'solves' to link this PR to an issue -->
+
+This PR solves #12345
+
+#### Pull request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
+- [ ] **Tests**: This PR includes tests for covering the features or bug fixes, or it includes an explanation of why it does not.
+- [ ] **Screenshots and/or logs**: This PR includes screenshots, videos or logs of the changes made, or an explanation of why it does not.
+- [ ] **Commits descriptions**: All commits in this PR contain a title and description according to [CONTRIBUTING.md](https://github.com/profusion/quickstart-nodejs-rest/blob/master/doc/CONTRIBUTING.md#commits).
+
+<!-- Also, don't forget to review your code before marking it as ready to merge -->
+
+### Progress
+
+<!-- If your PR is WIP, use checkboxes to show that you did and what you have to do. For example: -->
+
+- [x] New endpoint created;
+- [ ] Update organizations;
+- [ ] Create tests;
+
+### Description
+
+<!-- Describe your solution for the issue here. Adding videos, screenshots and expected logs may help the others to understand what is new.
+-->
```

Comment:
Description should 100% be the first topic, not the second to last

Reply:
Moved to the beginning of the template.